### PR TITLE
Add webpack-server-dev and mailhog

### PR DESCRIPTION
**Pull request by ylavoie**
Document devServer and MailHog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,48 +61,61 @@ Ten containers are created:
...
-## Creating JavaScript output
+### Creating JavaScript output
 
 To use the web-app, a "transpiled" version of the JavaScript code must
 be available in the `UI/js/` directory. This is created from `UI/js-src/`
 by running
 
 ```bash
-$ make dojo
+make dojo
```
Comment (ehuelsmann) [inline README.md:79]: This should be `make js` for 1.10+.

File: lsmb-dev [hunk 2]
```diff
@@ -74,6 +78,8 @@ show_result() {
 	== should be available at
 	======================================
 	host         : http://${IpAddr_host}:${HostPort}
+	host_dev     : http://${IpAddr_host}:${HostDevPort}
+	mailhog      : http://${IpAddr_host}:${MailHogPort}
```
Comment (ehuelsmann) [inline lsmb-dev:82]: I know you have a habit of working with the op of the host - which is different from my habit of working directly with the container IP. I have no idea if either is better, but isn't our list becoming rather inconsistent with the items at the top using one approach and the rest the other?

**Review by ehuelsmann — COMMENTED** [review]
Thanks for the documentation update!

Apart from one comment, this is very nice (thorough) maintenance.

File: lsmb-dev (hunk 2 above)
Comment (ylavoie) [reply lsmb-dev]: Fixed

**Pull request merged**